src/arrayHandling.ts

```diff
@@ -64,6 +65,15 @@ export function handleArrayLogic<Variables>(
                     return item
                 }
 
+                // handle properties of an object
+                if (typeof item === "object") {
+                    const newReference = `$${reference.substring(
+                        reference.indexOf("#.") + 1,
+                    )}`
+                    const found = findNamedChild(newReference, item)
```

Comment:
did you mean `options.findNamedChild`? the global FNC is not array-aware

Reply:
Yes, my bad, fixed in next commit.